### PR TITLE
Fix ra-input-rich-text dark theme compatibility 

### LINKED ISSUE #5112: Theming issue in ra-input-rich-text
**What you were expecting:**
Theming of `ra-input-rich-text` should be according to the theme passed in admin

**What happened instead:**
It's working partially. The only problem is that the bottom border in an inactive state is still black, making it nearly invisible for end-user.

**Steps to reproduce:**
1. Create a sample react admin, with a custom theme.
2. Add `type: dark` as the pallette option

**Related code:**
```js
const theme = createMuiTheme({
  palette: {
    primary: {
      light: '#ffb74d',
      main: '#ffb74d',
      dark: '#f57c00'
    },
    secondary: {
      light: '#81c784',
      main: '#2cb67d',
      dark: '#388e3c'
    },
    type: 'dark',
    background: {
      default: '#16161a'
    }
  }
});
```
```jsx
<Admin
    theme={theme}
    loginPage={MyLoginPage}
    layout={MainLayout}
    authProvider={authProvider}
    dashboard={Dashboard}
    dataProvider={dataProvider}
  >
    <Resource name='users' list={UserList} icon={UserIcon} />
  </Admin>
```

**Other information:**

Screenshot from 2020-08-04 19-47-45

**Environment**

* React-admin version: ^3.7.1
* React version: ^16.13.1
* Browser: Chrome Version 84.0

### PR DESCRIPTION
**Pull request by manishprivet**
Updated styles for ra-input-rich-text for better dark mode compatibility.

Fixes #5112

[EDIT]
Screenshots

Before:

Screenshot from 2020-08-04 20-22-42

After:

Screenshot from 2020-08-04 20-21-04

### CODE REVIEW

**Comment by djhi** [comment]
Thanks! Would you mind adding a screenshot in the PR description?

**Comment by manishprivet** [comment]
Yeah, Sure.

Before:

Screenshot from 2020-08-04 20-22-42

After:

Screenshot from 2020-08-04 20-21-04

**Comment by manishprivet** [comment]
> Thanks! Would you mind adding a screenshot in the PR description?

Done

**Comment by djhi** [comment]
Approved. You'll have to fix the eslint/prettier errors though

**Comment by djhi** [comment]
Thanks!

File: packages/ra-input-rich-text/package.json [hunk 1]
```diff
@@ -46,6 +46,7 @@
         "react-dom": "^16.9.0"
     },
     "dependencies": {
+        "@material-ui/core": "^4.11.0",
```
Comment (fzaninotto) [inline packages/ra-input-rich-text/package.json:49]: not good: the ra-ui-material-ui dependency is ^4.3.3. This is a breaking change. 

Why did you need an earlier version in the first place?

Comment (manishprivet) [reply packages/ra-input-rich-text/package.json:49]: I actually didn't change that. Linters were throwing error of `@material-ui/core` not being installed, So I installed it.
